ui/src/components/header/DomainDetails.js

```diff
@@ -386,6 +386,14 @@ class DomainDetails extends React.Component {
                         </ValueDiv>
                         <LabelDiv>AWS ACCOUNT ID</LabelDiv>
                     </SectionDiv>
+                    <SectionDiv>
+                        <ValueDiv>
+                            {this.props.domainDetails.gcpProject
+                                ? this.props.domainDetails.gcpProject
+                                : 'N/A'}
+                        </ValueDiv>
+                        <LabelDiv>GCP ACCOUNT ID</LabelDiv>
```

Comment:
This should be GCP PROJECT ID (no accounts in GCP)

Reply:
Fixed! New branch for this change: https://github.com/AthenZ/athenz/pull/2065